README.md

```diff
@@ -0,0 +1,14 @@
+# Todoly
+ `Todoly` is a task organizer
+
+## Usage
+1.Build <br/>
+
+```
+docker build -t <image name> .
```

Comment:
It doesn't need to be different name. One can copy and paste this from here and expect it to work. Something like this ;

```
docker build -t todoly:latest .
```